docs/steam.md

```diff
@@ -17,7 +17,7 @@ To manually start **Gaming Mode** (also known as the **Steam Deck UI**), there a
 
 ## Start On Boot
 
-To automatically launch Gaming Mode on boot and enable desktop switching from the power menu:
+The following configuration snippet configures the system to automatically launch Gaming Mode on boot, and enables desktop switching to Plasma 5.
```

Comment:
Maybe we should update this to Plasma 6.

Reply:
Can you share the exact config you'd ~~support~~ recommend for this? Just so I'm off the hook :).

Reply:
Just replace `services.xserver.desktopManager.plasma5.enable = true` with `services.desktopManager.plasma6.enable = true`